sympy/solvers/solvers.py

```diff
@@ -982,7 +982,7 @@ def _sympified_list(w):
             f[i] = fi
 
         if fi.is_Relational:
-            return reduce_inequalities(f, symbols=symbols)
+            return reduce_inequalities(f, dict_flag=flags.get('dict', False), symbols=symbols)
```

Comment:
They new method arguments must be added at the end. Also I do not see `reduce_inequalities` the order is correct. Please check